php/class-post-type.php

```diff
@@ -675,4 +711,68 @@ public function hide_disabled_publishing_actions( $post ) {
 		</style>
 		<?php
 	}
+
+	/**
+	 * Get conflicts settings
+	 *
+	 * @param \WP_Post $post post to compare conflict values.
+	 * @param array    $settings setting to search for optional.
+	 *
+	 * @return array
+	 */
+	public function get_conflicts_setting( $post, $settings = array() ) {
+		global $wpdb;
+		if ( $post ) {
+			$post = get_post( $post );
+		}
+		if ( empty( $settings ) ) {
+			$content = $this->get_post_content( $post );
+			$return = array();
+			if ( empty( $content ) || ! is_array( $content ) ) {
+				return $return;
+			}
+			$settings = array_keys( $content );
+			if ( empty( $settings ) ) {
+				return $return;
+			}
+		}
+		$return = array();
```

Comment:
I think `$return` should be renamed to something more descriptive, like `$conflicted_settings`.